doc/guide/plugin.rst

```diff
@@ -1221,11 +1221,30 @@ Test Your New Model
 Minimal Testing
 ...............
 
-From SasView either open the Python shell (*Tools* > *Python Shell/Editor*)
-or the plugin editor (*Fitting* > *Plugin Model Operations* > *Advanced
-Plugin Editor*), load your model, and then select *Run > Check Model* from
-the menu bar. An *Info* box will appear with the results of the compilation
-and a check that the model runs.
+In SasView 5.x, plugin models are loaded 'on-the-fly'. So to test a plugin
+go to a FitPage (or create a new one with *Fitting* > *New Fit Page*),
+select *Category* > *Plugin Models*, and then select your model from the
+*Model Name* dropdown. Then click the *Calculate* button. If the model
+compiles succesfully, two things will happen: the *Calculate* button will
+change to *Show Plot*, and in the *Log Explorer* window something like
+this will appear, possibly followed by a report of the unit test in your
+plugin model:
+
+11:06:37 - INFO: make python model peak_voigt
+
+**NB: If your plugin model does not appear in the dropdown, try opening a
+new FitPage, or restarting SasView.**
```

Comment:
If the plugin model doesn't appear in the dropdown then restarting sasview probably won't fix it.  It makes for a tedious testing cycle if you have to restart each time only to see that it is still failing to load your new model.

Please remove the comment, or if there are still bugs in model handling that sometimes make it necessary, move it below a quick check for errors in `~/sasview.log` and `.sasview/plugin_models/plugins.log`.

The old instructions let you test the model without leaving the editor, which is a much quicker cycle. Assuming the method still works, it should be listed as the first thing to check when creating a new model.